roles/confluent.variables/vars/main.yml

```diff
@@ -394,6 +394,7 @@ schema_registry_properties:
       debug: 'false'
       schema.registry.group.id: schema-registry
       kafkastore.topic: _schemas
+      kafkastore.topic.replication.factor: "{{ [ groups['kafka_broker'] | default(['localhost']) | length, default_internal_replication_factor ] | min }}"
```

Comment:
should we create a `schema_registry_default_internal_replication_factor` var then, seems like thats the pattern w the other components?

Reply:
Yeap, make sense. Will add it